web/src/scheduled_messages.js

```diff
@@ -18,7 +18,8 @@ import * as timerender from "./timerender";
 export const MINIMUM_SCHEDULED_MESSAGE_DELAY_SECONDS = 5 * 60;
 export const SCHEDULING_MODAL_UPDATE_INTERVAL_IN_MILLISECONDS = 60 * 1000;
 
-export let scheduled_messages_data = [];
+export let scheduled_messages_data = {};
+// scheduled_messages_data is a dictionary where key=timestamp and value=scheduled_messages
```

Comment:
Why you think `timestamp` is a good key here? Also, this message should be above the variable.